src/main/java/org/fogbowcloud/manager/core/threads/AttendSpawningOrdersThread.java

```diff
@@ -0,0 +1,129 @@
+package org.fogbowcloud.manager.core.threads;
+
+import java.util.Map;
+
+import org.apache.log4j.Logger;
+import org.fogbowcloud.manager.core.datastructures.OrderStateTransitioner;
+import org.fogbowcloud.manager.core.datastructures.SharedOrderHolders;
+import org.fogbowcloud.manager.core.exceptions.OrderStateTransitionException;
+import org.fogbowcloud.manager.core.models.orders.Order;
+import org.fogbowcloud.manager.core.models.orders.OrderState;
+import org.fogbowcloud.manager.core.models.orders.OrderType;
+import org.fogbowcloud.manager.core.models.orders.instances.ComputeOrderInstance;
+import org.fogbowcloud.manager.core.models.orders.instances.InstanceState;
+import org.fogbowcloud.manager.core.models.orders.instances.OrderInstance;
+import org.fogbowcloud.manager.core.plugins.ComputePlugin;
+import org.fogbowcloud.manager.core.utils.SshConnectivityUtil;
+import org.fogbowcloud.manager.core.utils.TunnelingServiceUtil;
+
+public class AttendSpawningOrdersThread extends Thread {
```

Comment:
I do not think that the name of this Thread should be AttendSpawningOrdersThread, check the document contract. I think that is SpawningProcessor. Therefore, check this [PR](https://github.com/fogbow/fogbow-manager-core/pull/13/files) with the OpenProcessor Thread implementation revised.

Reply:
Thanks for the tip, of according to the document contract, this is must be  SpawningMonitor...